### ISSUE TITLE
Reconcile renditions

### ISSUE THREAD

**Issue by joewiz** [HistoryAtState/frus#272]
Now that we've finished digitizing the print catalog, we should begin consolidating the variations in `<rendition>` values used across the TEI edition. Here is the standard markup used in most volumes now:

```
<encodingDesc>
    <styleDefDecl scheme="css" schemeVersion="2.1"/>
    <tagsDecl>
        <rendition xml:id="top">vertical-align: top;</rendition>
        <rendition xml:id="bottom">vertical-align: bottom;</rendition>
        <rendition xml:id="middle">vertical-align: middle;</rendition>
        <rendition xml:id="nowrap">white-space: nowrap !important;</rendition>
        <rendition xml:id="nopad">padding-right: 0.5em !important;</rendition>
        <rendition xml:id="border-top">border-top: 1px black solid;</rendition>
        <rendition xml:id="border-bottom">border-bottom: 1px black solid;</rendition>
        <rendition xml:id="border-left">border-left: 1px black solid;</rendition>
        <rendition xml:id="border-right">border-right: 1px black solid;</rendition>
        <rendition xml:id="center">display: block; text-align: center;</rendition>
        <rendition xml:id="center-block">margin-left: auto; margin-right: auto;</rendition>
        <rendition xml:id="underline">text-decoration: underline;</rendition>
        <rendition xml:id="hanging">padding-left:1.5em; text-indent:-1.5em;</rendition>
        <rendition xml:id="left">display: inline; float: left;</rendition>
        <rendition xml:id="right">display: inline; float: right;</rendition>
    </tagsDecl>
    ...
```

There are some volumes with missing, variant, or new entries. Below is a report showing these. 

@vak2ve As part of our FRUS visual style guide project, would you please review the cases below and send me your recommendations? The cases of volumes with missing entries is pretty straightforward to resolve (just add the missing entries in). For cases where the rendition was different than expected, please suggest whether to switch the value to the version above, or whether we should take some step to preserve that value for the volume. The idea is to create a common rendition set that all volumes will reference and include. We can still have per-volume overrides/additions, but only when it's needed. 

(After we review the rendition values, let's take a look at the `@rend` values and move as many `@rend` cases to proper `@rendition` definitions.)

```json
[
    {
        "frus1865p4": {"center-block": "missing"}
    },
    {
        "frus1892": {
            "new-entries": {"inline": "display: inline;"}
        }
    },
    {
        "frus1898": {
            "new-entries": {"inline": "display: inline;"}
        }
    },
    {
        "frus1914Supp": {"center-block": "missing"}
    },
    {
        "frus1917-72PubDipv07": {
            "new-entries": {"image-center": "margin-left: auto; margin-right: auto;"},
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1917-72PubDipv08": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1922v01": {
            "new-entries": {"inline": "display: inline;"}
        }
    },
    {
        "frus1944v03": {
            "new-entries": {"border-bottom-double": "border-bottom: 1px black solid; border-bottom-style: double;"}
        }
    },
    {
        "frus1944v04": {
            "new-entries": {"border-bottom-double": "border-bottom: 1px black solid; border-bottom-style: double;"}
        }
    },
    {
        "frus1945Malta": {"center-block": "missing"}
    },
    {
        "frus1947v04": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            }
        }
    },
    {
        "frus1947v05": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            }
        }
    },
    {
        "frus1948v01p1": {"center-block": "missing"}
    },
    {
        "frus1948v01p2": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": "missing"
        }
    },
    {
        "frus1948v02": {"center-block": "missing"}
    },
    {
        "frus1948v03": {"center-block": "missing"}
    },
    {
        "frus1948v04": {"center-block": "missing"}
    },
    {
        "frus1948v05p1": {"center-block": "missing"}
    },
    {
        "frus1948v05p2": {"center-block": "missing"}
    },
    {
        "frus1948v06": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": "missing"
        }
    },
    {
        "frus1948v07": {"center-block": "missing"}
    },
    {
        "frus1948v09": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": "missing"
        }
    },
    {
        "frus1949v01": {
            "new-entries": {"border-bottom-double": "border-bottom: 1px black solid; border-bottom-style: double;"},
            "center-block": "missing"
        }
    },
    {
        "frus1949v02": {"center-block": "missing"}
    },
    {
        "frus1949v03": {"center-block": "missing"}
    },
    {
        "frus1949v04": {"center-block": "missing"}
    },
    {
        "frus1949v05": {"center-block": "missing"}
    },
    {
        "frus1949v06": {"center-block": "missing"}
    },
    {
        "frus1949v07p1": {"center-block": "missing"}
    },
    {
        "frus1949v07p2": {"center-block": "missing"}
    },
    {
        "frus1949v08": {"center-block": "missing"}
    },
    {
        "frus1949v09": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": {
                "found": "display: block; margin-left: auto; margin-right: auto;",
                "expected": "margin-left: auto; margin-right: auto;"
            }
        }
    },
    {
        "frus1950-55Intel": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": "missing"
        }
    },
    {
        "frus1950v03": {"center-block": "missing"}
    },
    {
        "frus1950v04": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            }
        }
    },
    {
        "frus1950v05": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": "missing"
        }
    },
    {
        "frus1950v06": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": "missing"
        }
    },
    {
        "frus1950v07": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": "missing"
        }
    },
    {
        "frus1951-54Iran": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1951-54IranEd2": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1951v02": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            }
        }
    },
    {
        "frus1952-54v12p1": {"center-block": "missing"}
    },
    {
        "frus1952-54v12p2": {"center-block": "missing"}
    },
    {
        "frus1952-54v13p1": {"center-block": "missing"}
    },
    {
        "frus1952-54v13p2": {"center-block": "missing"}
    },
    {
        "frus1952-54v14p1": {"center-block": "missing"}
    },
    {
        "frus1952-54v14p2": {"center-block": "missing"}
    },
    {
        "frus1952-54v15p1": {"center-block": "missing"}
    },
    {
        "frus1952-54v15p2": {"center-block": "missing"}
    },
    {
        "frus1952-54v16": {"center-block": "missing"}
    },
    {
        "frus1955-57v01": {"center-block": "missing"}
    },
    {
        "frus1955-57v02": {"center-block": "missing"}
    },
    {
        "frus1955-57v03": {"center-block": "missing"}
    },
    {
        "frus1955-57v04": {"center-block": "missing"}
    },
    {
        "frus1955-57v05": {"center-block": "missing"}
    },
    {
        "frus1955-57v06": {"center-block": "missing"}
    },
    {
        "frus1955-57v07": {"center-block": "missing"}
    },
    {
        "frus1955-57v08": {"center-block": "missing"}
    },
    {
        "frus1955-57v09": {"center-block": "missing"}
    },
    {
        "frus1955-57v10": {"center-block": "missing"}
    },
    {
        "frus1955-57v11": {"center-block": "missing"}
    },
    {
        "frus1955-57v12": {"center-block": "missing"}
    },
    {
        "frus1955-57v13": {"center-block": "missing"}
    },
    {
        "frus1955-57v14": {"center-block": "missing"}
    },
    {
        "frus1955-57v15": {"center-block": "missing"}
    },
    {
        "frus1955-57v16": {"center-block": "missing"}
    },
    {
        "frus1955-57v17": {"center-block": "missing"}
    },
    {
        "frus1955-57v18": {"center-block": "missing"}
    },
    {
        "frus1955-57v19": {"center-block": "missing"}
    },
    {
        "frus1955-57v20": {"center-block": "missing"}
    },
    {
        "frus1955-57v21": {"center-block": "missing"}
    },
    {
        "frus1955-57v22": {"center-block": "missing"}
    },
    {
        "frus1955-57v23p1": {"center-block": "missing"}
    },
    {
        "frus1955-57v23p2": {"center-block": "missing"}
    },
    {
        "frus1955-57v24": {"center-block": "missing"}
    },
    {
        "frus1955-57v25": {"center-block": "missing"}
    },
    {
        "frus1955-57v26": {"center-block": "missing"}
    },
    {
        "frus1955-57v27": {"center-block": "missing"}
    },
    {
        "frus1958-60v01": {"center-block": "missing"}
    },
    {
        "frus1958-60v02": {"center-block": "missing"}
    },
    {
        "frus1958-60v03": {"center-block": "missing"}
    },
    {
        "frus1958-60v04": {"center-block": "missing"}
    },
    {
        "frus1958-60v05": {"center-block": "missing"}
    },
    {
        "frus1958-60v06": {"center-block": "missing"}
    },
    {
        "frus1958-60v07p1": {"center-block": "missing"}
    },
    {
        "frus1958-60v07p2": {"center-block": "missing"}
    },
    {
        "frus1958-60v08": {"center-block": "missing"}
    },
    {
        "frus1958-60v09": {"center-block": "missing"}
    },
    {
        "frus1958-60v10p1": {"center-block": "missing"}
    },
    {
        "frus1958-60v10p2": {"center-block": "missing"}
    },
    {
        "frus1958-60v11": {"center-block": "missing"}
    },
    {
        "frus1958-60v12": {"center-block": "missing"}
    },
    {
        "frus1958-60v13": {"center-block": "missing"}
    },
    {
        "frus1958-60v14": {"center-block": "missing"}
    },
    {
        "frus1958-60v15": {"center-block": "missing"}
    },
    {
        "frus1958-60v16": {"center-block": "missing"}
    },
    {
        "frus1958-60v17": {"center-block": "missing"}
    },
    {
        "frus1958-60v18": {"center-block": "missing"}
    },
    {
        "frus1958-60v19": {"center-block": "missing"}
    },
    {
        "frus1961-63v07-09mSupp": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1969-76v10": {
            "new-entries": {
                "inline-float-left": "display: inline; float: left;",
                "inline-float-right": "display: inline; float: right;"
            }
        }
    },
    {
        "frus1969-76v23": {
            "hanging": {
                "found": "padding-left:2em; text-indent:-1.5em;",
                "expected": "padding-left:1.5em; text-indent:-1.5em;"
            },
            "center-block": "missing"
        }
    },
    {
        "frus1969-76ve05p1": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1969-76ve09p2": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1969-76ve11p2": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1969-76ve14p2": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    },
    {
        "frus1969-76ve16": {
            "border-left": "missing",
            "nowrap": "missing",
            "bottom": "missing",
            "underline": "missing",
            "border-bottom": "missing",
            "hanging": "missing",
            "nopad": "missing",
            "border-top": "missing",
            "center": "missing",
            "center-block": "missing",
            "top": "missing",
            "middle": "missing",
            "border-right": "missing"
        }
    }
]
```

**Comment by vak2ve** [comment]
A lot of the aberrations here seem to be related to volume-specific table work, with center-block for table alignment and a deeper hanging indentation (2em instead of the standard 1.5) for irregular cells in participant tables unique to 1940s volumes. A few examples:

https://history.state.gov/historicaldocuments/frus1947v04/pg_852
<img width="337" alt="Screen Shot 2019-12-04 at 1 21 53 PM" src="https://user-images.githubusercontent.com/8865513/70169613-27b1ea00-1699-11ea-8b24-f8afba1c7045.png">

https://history.state.gov/historicaldocuments/frus1948v01p1/pg_403
<img width="332" alt="Screen Shot 2019-12-04 at 1 24 48 PM" src="https://user-images.githubusercontent.com/8865513/70169789-7fe8ec00-1699-11ea-99f9-a1de6aded712.png">

However by the 1950s volumes the indentation changed to the expected standard for a hanging value, and should match the standard markup in most volumes.

https://history.state.gov/historicaldocuments/frus1950v07/pg_263
<img width="415" alt="Screen Shot 2019-12-04 at 1 27 47 PM" src="https://user-images.githubusercontent.com/8865513/70170008-e706a080-1699-11ea-9425-9af0f7cb956a.png">

https://history.state.gov/historicaldocuments/frus1951v02/pg_140
<img width="408" alt="Screen Shot 2019-12-04 at 1 31 39 PM" src="https://user-images.githubusercontent.com/8865513/70170299-6f854100-169a-11ea-8194-424e89187cf3.png">

In short, all hanging rendition values after 1949v09 should be as expected, and 1947v04, 1947v05, 1948v01p2, 1948v06, 1948v07, 1948v09, and 1949v09 should keep the unique 2em version. Since the participant table format was the same, I must have copied the early version of rendition values without checking if the pattern still held in later volumes. Thanks for that catch!

The missing values for the rest should all be added, I think. Even in volumes where the center-block rendition value isn't applied (either because no tables with that alignment exist or because I haven't gotten around to marking them as such), it doesn't hurt to have it listed, and it might be applicable down the road.

Does that answer all the ambiguities?

**Comment by joewiz** [comment]
@vak2ve Thank you! I'll give this a closer look ASAP and will let you know if I have any questions.
